core/src/main/res/values/strings.xml

```diff
@@ -402,102 +402,102 @@
+    <string name="pref_parallel_downloads_title">Concurrent Downloads</string>
     <string name="pref_episode_cache_title">Episode Cache</string>
-    <string name="pref_episode_cache_summary">Total number of downloaded episodes cached on the device. Automatic download will be suspended if this number is reached.</string>
+    <string name="pref_episode_cache_summary">Max. number of downloaded episodes to cache on the device. Auto-downloading is turned off when it is reached.</string>
     <string name="pref_episode_cover_title">Use Episode Cover</string>
     <string name="pref_episode_cover_summary">Use the episode specific cover in lists whenever available. If unchecked, the app will always use the podcast cover image.</string>
     <string name="pref_show_remain_time_title">Show Remaining Time</string>
     <string name="pref_show_remain_time_summary">Display remaining time of episodes when checked. If unchecked, display total duration of episodes.</string>
     <string name="pref_theme_title_use_system">Use system theme</string>
     <string name="pref_theme_title_light">Light</string>
     <string name="pref_theme_title_dark">Dark</string>
-    <string name="pref_theme_title_trueblack">Black (AMOLED ready)</string>
+    <string name="pref_theme_title_trueblack">Black (AMOLED)</string>
     <string name="pref_episode_cache_unlimited">Unlimited</string>
-    <string name="pref_gpodnet_authenticate_title">Login</string>
-    <string name="pref_gpodnet_authenticate_sum">Login with your gpodder.net account in order to sync your subscriptions.</string>
-    <string name="pref_gpodnet_logout_title">Logout</string>
-    <string name="pref_gpodnet_logout_toast">Logout was successful</string>
+    <string name="pref_gpodnet_authenticate_title">Log in</string>
+    <string name="pref_gpodnet_authenticate_sum">Log in with your gpodder.net account to sync your subscriptions.</string>
+    <string name="pref_gpodnet_logout_title">Log out</string>
+    <string name="pref_gpodnet_logout_toast">Logged out</string>
     <string name="pref_gpodnet_setlogin_information_title">Change login information</string>
-    <string name="pref_gpodnet_setlogin_information_sum">Change the login information for your gpodder.net account.</string>
-    <string name="pref_gpodnet_sync_changes_title">Synchronize now</string>
+    <string name="pref_gpodnet_setlogin_information_sum">Change gpodder.net login info.</string>
+    <string name="pref_gpodnet_sync_changes_title">Sync…</string>
     <string name="pref_gpodnet_sync_changes_sum">Sync subscription and episode state changes with gpodder.net.</string>
     <string name="pref_gpodnet_full_sync_title">Force full synchronization</string>
     <string name="pref_gpodnet_full_sync_sum">Sync all subscriptions and episode states with gpodder.net.</string>
     <string name="pref_gpodnet_login_status"><![CDATA[Logged in as <i>%1$s</i> with device <i>%2$s</i>]]></string>
```

Comment:
> What is "%2$s" here?

`%2$s` here refers to the device name, if I'm not mistaken, so in gpodder you can identify in the logs from which device the log-in happened.